bindings/pyroot/pythonizations/test/rtensor.py

```diff
@@ -30,7 +31,9 @@ def test_dtypes(self):
             root_obj = ROOT.TMVA.Experimental.AsRTensor(np_obj)
             self.assertTrue(check_shape(root_obj, np_obj))
             np_obj[0,0] = 42
-            self.assertTrue(root_obj[0,0] == 42)
+            # disable this test on Win64, it crashes
+            if not 'win32' in sys.platform or sys.maxsize < 2**32:
+                self.assertTrue(root_obj[0,0] == 42)
```

Comment:
So there's no way to make this test work?
On the other hand, the condition does not really disable it for win64, does it?

Reply:
> So there's no way to make this test work?

Didn't find a way and don't really know how to debug this...



> On the other hand, the condition does not really disable it for win64, does it?

Yes, it does `sys.maxsize` is `>2**32` (`2**63`)

Reply:
Does the issue appear with a particular data type? If so, which one?

Reply:
> Does the issue appear with a particular data type? If so, which one?

I'll try to figure out

Reply:
All data types

Reply:
Ok, you don't think it's worth trying to understand the issue? Is the failure in the access (`root_obj[0,0]`)? Or is it the assertion that fails because the extracted value is not 42?

Since it is a TMVA feature, @lmoneta might want to give his opinion.

Reply:
OK, thanks, I'll keep trying to debug this.

Reply:
But the changes in [RTensorPyz.cxx](https://github.com/root-project/root/pull/9964/files#diff-6e43179dc9f4929d5ec93caff48bd1cb22ed2afbe73c4470b60cab605d1f3b4e) are still required!

Reply:
Yes indeed, you want to move those to another PR so we can merge them?

Reply:
Yes, will do